reference/hardware/bump-switches.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: reference
+title: Bump Switches, Drake
```

Comment:
What's Drake? If it's the name of the board, maybe in parenthesis, or as a tag? This title will eventually appear in a navigation bar somewhere

Reply:
Actually, based on the double backtick links in other files, I'd say we should remove the board name (put it under tags) and use "Bump Switches" and "Low Power Board", etc so the linking stuff can find the appropriate page